com.unity.multiplayer.mlapi/Tests/Runtime/Metrics/WaitForMetricValues.cs

```diff
@@ -62,9 +89,9 @@ public void Observe(MetricCollection collection)
             }
         }
 
-        private IEnumerator Wait(uint maxNbFrames)
+        public IEnumerator Wait(uint maxNbFrames)
```

Comment:
This is the only intentional change as I needed to expose Wait() for the tests.

Reply:
That works!